### PR TITLE
Checkstyle

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

File: config/google-checks-no-indent.xml [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+            <property name="allowedAbbreviationLength" value="1"/>
+        </module>
+        <module name="OverloadMethodsDeclarationOrder"/>
+        <module name="VariableDeclarationUsageDistance"/>
+        <module name="CustomImportOrder">
+            <property name="sortImportsInGroupAlphabetically" value="true"/>
+            <property name="separateLineBetweenGroups" value="true"/>
+            <property name="customImportOrderRules" value="STATIC###THIRD_PARTY_PACKAGE"/>
+        </module>
+        <module name="MethodParamPad"/>
```
Comment (coollog) [inline config/google-checks-no-indent.xml]: Why was

        <module name="NoWhitespaceBefore">
           <property name="tokens" value="COMMA, SEMI, POST_INC, POST_DEC, DOT, ELLIPSIS, METHOD_REF"/>
            <property name="allowLineBreaks" value="true"/>
        </module>

removed?

Comment (coollog) [reply config/google-checks-no-indent.xml]: Looks like the google-java-format plugin takes care of this. Maybe add this back with the same format as the Indentation module above just for consistency?

Comment (loosebazooka) [reply config/google-checks-no-indent.xml]: I don't recall removing NoWhitespaceBefore? What do you mean?

Comment (coollog) [reply config/google-checks-no-indent.xml]: Sorry I was referring to the original file: https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml

Diffing with this -no-indent file gives:

```
167c167
<         <module name="Indentation">
---
>         <!-- delegate to google-java-format module name="Indentation">
174c174
<         </module>
---
>         </module-->
187,190d186
<         <module name="NoWhitespaceBefore">
<           <property name="tokens" value="COMMA, SEMI, POST_INC, POST_DEC, DOT, ELLIPSIS, METHOD_REF"/>
<             <property name="allowLineBreaks" value="true"/>
<         </module>
```

Comment (loosebazooka) [reply config/google-checks-no-indent.xml]: hah, last edited 11 hours ago, I must have missed it when I copied it over, lemme see if they fixed that annotation issue too.

Comment (loosebazooka) [reply config/google-checks-no-indent.xml]: updated.

**Pull request merged**